models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php

```diff
@@ -201,7 +178,7 @@ public function updateStatus($taskId, $newStatus, $prevStatus = null)
     /**
      * @inheritdoc
      */
-    public function addReport($taskId, Report $report, $newStatus = null)
+    public function addReport(string $taskId, Report $report, ?string $newStatus = null): int
```

Comment:
```suggestion
    public function addReport(string $taskId, Report $report, string $newStatus = null): int
```

Default value already provided